browser/brave_shields/brave_shields_web_contents_observer.cc

```diff
@@ -249,11 +255,15 @@ void BraveShieldsWebContentsObserver::ReadyToCommitNavigation(
     }
   }
 
-  auto render_frame_hosts = navigation_handle->GetWebContents()->GetAllFrames();
-  for (content::RenderFrameHost* rfh : render_frame_hosts) {
-    GetBraveShieldsRemote(rfh)->SetAllowScriptsFromOriginsOnce(
-        allowed_script_origins_);
-  }
+  navigation_handle->GetWebContents()->ForEachRenderFrameHost(
```

Comment:
are you sure we don't need `->GetMainFrame()` ?

Reply:
@mariospr, I wonder about the opposite actually for the prior change (line 62). In https://bugs.chromium.org/p/chromium/issues/detail?id=1208438 it says: 
<pre>
Most migrations should be fairly mechanical. For example:
...
  for (content::RenderFrameHost* rfh : web_contents()->GetAllFrames()) {
    DoSomething(rfh);
  }
would both become
  web_contents()->ForEachRenderFrameHost(
      base::BindRepeating(&MyClass::DoSomething,
                          base::Unretained(this)));
</pre>
**if all frames including non-primary are needed. Or**
<pre>
web_contents()->GetMainFrame()->ForEachRenderFrameHost(
      base::BindRepeating(&MyClass::DoSomething,
                          base::Unretained(this)));
</pre>
**if only primary pages are needed.**

Reply:
@iefremov I think in this case it's ok because we want all the RenderFrameHosts, not just the ones for the main frame.

However, I think @mkarolin is right on that the translation might not have been correct up in line 62 (for `pdateContentSettingsToRendererFrames()`), I'll fix that one (and re-review the entire commit)

Reply:
Fixed the suggestion by Max in https://github.com/brave/brave-core/commit/2827adb83f4bd7ad66a9f10b74dd85cdfa3a2f38